pkg/client/openshift/buildconfig.go

```diff
@@ -132,10 +132,13 @@ func (b *buildConfig) TriggerBuild(bc *buildv1.BuildConfig, triggeredBy string)
 
 // TriggerBuildFromFile will be called by kogito-cli when a build from file is performed.
 // When called a new build will be triggered with the request kogito resource or a tgz file.
-func (b *buildConfig) TriggerBuildFromFile(namespace string, bodyPost io.Reader, options *buildv1.BinaryBuildRequestOptions) (*buildv1.Build, error) {
-
+func (b *buildConfig) TriggerBuildFromFile(namespace string, bodyPost io.Reader, options *buildv1.BinaryBuildRequestOptions, binaryBuild bool) (*buildv1.Build, error) {
 	result := &buildv1.Build{}
-	buildName := fmt.Sprintf("%s%s", options.Name, "-builder")
+
+	buildName := options.Name
+	if !binaryBuild {
+		buildName += "-builder"
```

Comment:
`builder` is a constant for the KogitoBuild instance? I believe you can take that from the `build` package:
https://github.com/kiegroup/kogito-cloud-operator/blob/master/pkg/controller/kogitobuild/build/utils.go#L26

You might refactor this to be public.

Reply:
I get a cyclical import when trying to import from the `build` package:
```
import cycle not allowed
package main
	imports github.com/kiegroup/kogito-cloud-operator/pkg/framework
	imports github.com/kiegroup/kogito-cloud-operator/pkg/client/openshift
	imports github.com/kiegroup/kogito-cloud-operator/pkg/controller/kogitobuild/build
	imports github.com/kiegroup/kogito-cloud-operator/pkg/client/openshift
import cycle not allowed
package main
	imports github.com/kiegroup/kogito-cloud-operator/pkg/framework
	imports github.com/kiegroup/kogito-cloud-operator/pkg/client/openshift
	imports github.com/kiegroup/kogito-cloud-operator/pkg/controller/kogitobuild/build
	imports github.com/kiegroup/kogito-cloud-operator/pkg/framework
```

Reply:
You can export the constant to the `infrastructure` package and import it in both packages: https://docs.jboss.org/kogito/release/latest/html_single/#_kogito_operator_core_package_structure

Reply:
There was another once I moved it to `infrastructure`...
```
import cycle not allowed
package main
	imports github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure
	imports github.com/kiegroup/kogito-cloud-operator/pkg/framework
	imports github.com/kiegroup/kogito-cloud-operator/pkg/client/openshift
	imports github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure
```

Reply:
The constant is in the infra package and you can't use from the `cmd`? this is weird. Nevertheless, let it be then. We are passing through a major refactoring, I'll update later.

Reply:
The file we're trying to import from is in `pkg`.